vector/src/main/res/values/strings_no_weblate.xml

```diff
@@ -10,4 +10,6 @@
     <!-- Value MUST have 4 letters and MUST be in this list: https://www.unicode.org/iso15924/iso15924-codes.html. Example:  "Arab", "Cyrl", "Latn", etc. -->
     <string name="resources_script">Latn</string>
 
+    <string name="cut_the_slack_from_teams">Cut the slack from teams.</string>
```

Comment:
Maybe better to put it in [this file](https://github.com/vector-im/element-android/blob/main/vector/src/main/res/values/donottranslate.xml) and add the `translatable="false"` attribute

Reply:
will do :+1:

Reply:
updated https://github.com/vector-im/element-android/pull/4880/commits/2958644a46635287914105b4d4a988cbed6e4260